src/edfi-paging-test/edfi_paging_test/api/paginated_result.py

```diff
@@ -31,24 +31,22 @@ def __init__(
         self,
         resource_name: str,
         page_size: int,
-        api_response: Dict[str, Any],
+        api_response: List[Dict[str, Any]],
+        status_code: int,
         current_page: int = 1,
     ):
         self.page_size = page_size
         self.current_page = current_page
 
         if api_response is None:
-            self._current_page_items: Dict[str, Any] = []
+            self.current_page_items: List[Dict[str, Any]] = []
         else:
-            self._current_page_items = api_response
+            self.current_page_items = api_response
 
         self._api_response = api_response
         self._resource_name = resource_name
-
-    @property
-    def current_page_items(self) -> Dict[str, Any] :
-        return self._current_page_items
```

Comment:
Properties like this are actually not common in Python, since all of the variables are public anyway.